securityscanutils/commands/scan_repo.go

```diff
@@ -70,6 +75,10 @@ func doScanRepo(ctx context.Context, opts *scanRepoOptions) error {
 	if err != nil {
 		return err
 	}
+	additionalContext, err := fileutils.ReadFileString(opts.additionalContextFile)
```

Comment:
How does this behave when the additionalContextFile is empty, which is the default behavior? I think it might be helpful to have a simple unit test on this utility which shows the behaviors.